clients/docs/includes/configs/cloud/java-sr.config

```diff
@@ -1,10 +1,13 @@
-# Kafka
+# Kafka client (producer/consumer/admin)
```

Comment:
reconnect.backoff.max.ms defaults to 1 sec in AK. In CCloud, we have seen frequent reconnects due to bad user/passwd, which increases the load on the broker. I wonder if we should change the default to sth larger.

Reply:
Sounds good. What would you suggest? 5 seconds? Also, maybe we want to change the AK default as part of https://cwiki.apache.org/confluence/display/KAFKA/KIP-580%3A+Exponential+Backoff+for+Kafka+Clients

Reply:
Since this should be rare, maybe even setting it to 30 secs. Yes, it would be useful to bump up the defaults in AK too.